docs/versionNames.md

```diff
@@ -0,0 +1,41 @@
+
+## Fybrik Version Policy
+Fybrik uses [semantic versioning model](https://semver.org), which is the module adopted by a wide range of projects,
+e.g. Kubernetes, goLang, Helm and many others.
+
+Fybrik versions are of the form "vX.Y.Z" where X is the major version number, Y is the minor version number 
+and Z is the patch release number. 
+- Major Releases - released less frequently, indicate that **backward-incompatible public API changes**. This release 
+carries no guarantee that it will be backward compatible with preceding major versions.
+- Minor Releases - indicate **backward-compatible public API changes**. This release guarantees backward compatibility 
+and stability.
+- Patch Releases - should not change main functionality, mostly for security or bug fixes, and therefore guarantee 
+backward compatibility.
```

Comment:
Same as previous comment

Reply:
If you mean backward compatibility, it is irrelevant for path releases. They should not change Fybrik behavior, so there is no backward compatibility issue. It is the same for the current releases, e.g. there are v0.6.0 - v0.6.8 releases, they are all compatible.